EIPS/eip-nfr.md

```diff
@@ -0,0 +1,395 @@
+---
+eip: eip-nfr.md
+title: NFT Future Rewards (nFR) Standard
+description: In this EIP, we propose a multigenerational reward mechanism that rewards‌ all ‌owners of non-fungible tokens (NFT).
+author: Yale ReiSoleil <yale@iob.fi>, dRadiant (@dRadiant), D Wang, PhD <david@iob.fi>
+discussions-to: https://github.com/ethereum/EIPs/pull/5173
```

Comment:
This must be a link to a thread on Ethereum Magicians.

Reply:
That's right. Changed.